dbscripts/xml/install.xml

```diff
@@ -39,6 +39,7 @@
 	<migration class="PKP\migration\install\TombstoneMigration" />
 	<migration class="PKP\migration\install\DoiMigration" />
 	<migration class="PKP\migration\install\InstitutionsMigration" />
+	<migration class="PKP\migration\install\ReviewerSuggestionsMigration" />
```

Comment:
For now , this is only a place holder as this should be a part of `3.6.0` upgrade process .